apps/devhub/tests/test_views.py

```diff
@@ -3341,14 +3365,14 @@ def setUp(self):
         self.client.login(username='admin@mozilla.com', password='password')
 
     def test_bad_password(self):
-        r = self.client.post(self.url, dict(password='turd'))
+        r = self.client.post(self.url, dict(slug='nope'))
```

Comment:
Heh, I was a little surprised when I saw this. Not that it's particularly bad but not what I was expecting to see.